lib/Authen/SASL/Perl.pod

```diff
@@ -135,18 +157,25 @@ L<Authen::SASL::Perl::DIGEST_MD5>,
 L<Authen::SASL::Perl::EXTERNAL>,
 L<Authen::SASL::Perl::GSSAPI>,
 L<Authen::SASL::Perl::LOGIN>,
-L<Authen::SASL::Perl::PLAIN>
+L<Authen::SASL::Perl::OAUTHBEARER>,
+L<Authen::SASL::Perl::PLAIN>,
+L<Authen::SASL::Perl::XOAUTH2>
 
 =head1 AUTHOR
 
 Peter Marschall <peter@adpm.de>
 
+Aditya Garg <gargaditya08@live.com>
```

Comment:
It's customary to create a "CONTRIBUTORS" section for Perl modules and leave the original author in place. I suggest:

```suggestion
=head1 CONTRIBUTORS

Aditya Garg <gargaditya08@live.com>
```

Reply:
I am always confused with these things. Some projects want the person who edited to be named, some don't. Some have other rules.

I can even remove my name if you want.